modules/qr-code/backend/modules/qr-code/viewsets.py

```diff
@@ -0,0 +1,27 @@
+from rest_framework.views import APIView
+from rest_framework.response import Response
+from rest_framework import status
+import io
+import qrcode
+import base64
+
+
+class QRCodeView(APIView):
+
+    def post(self, request, *args, **kwargs):
+        qr = qrcode.QRCode(
+            version=1,
+            error_correction=qrcode.constants.ERROR_CORRECT_H,
+            box_size=4,
+            border=4,
+        )
+
+        qr.add_data(request.data['text'])
+        qr.make(fit=True)
+        img = qr.make_image()
```

Comment:
If customization is enabled, here it should contemplate foreground and background for matching the app theme